web/src/screens/private/Dashboard.js

```diff
@@ -3,38 +3,120 @@ import PropTypes from 'prop-types';
 
 import CURRENT_USER_QUERY from '../../graphql/current-user.query';
 
 import styles from './Dashboard.styles';
 
-const Dashboard = ({ classes, loading }) => {
+
+const displayRequests = (classes, user) => {
+  if (!user.groups.length) {
+    return (
+      <Paper className={classes.paper}>
+        <Typography component="p" align="center">
+          You are currently not assigned to any groups. <Link to="/groups">Click here to join groups</Link>
+        </Typography>
+      </Paper>
+    );
+  }
+
+  return (
+    <Paper className={classes.paper}>
+      <Toolbar className={classes.tableToolbar}>
+        <Typography variant="title">Requests</Typography>
+      </Toolbar>
+      <Table className={classes.table}>
+        <TableHead>
+          <TableRow>
+            <TableCell>Name</TableCell>
+            <TableCell>Group</TableCell>
+            <TableCell>Type</TableCell>
+            <TableCell>Start Date</TableCell>
+            <TableCell>End Date</TableCell>
+          </TableRow>
+        </TableHead>
+        <TableBody>
+          {user.schedules.map(schedule => (
+            <TableRow key={schedule.id}>
+              <TableCell>
+                <Link to={`/schedules/${schedule.id}`}>{schedule.name}</Link>
+              </TableCell>
+              <TableCell>{schedule.group.name}</TableCell>
+              {/* TODO: Make sure schedule object has this in the future */}
+              {/* <TableCell>{schedule.type}</TableCell> */}
+              <TableCell>
+                {schedule.startTime === 0 ? '-' : moment.unix(schedule.startTime).format('LLL')}
+              </TableCell>
+              <TableCell>
+                {schedule.endTime === 2147483647
+                  ? '-'
+                  : moment.unix(schedule.endTime).format('LLL')}
+              </TableCell>
+            </TableRow>
+          ))}
+        </TableBody>
+      </Table>
+    </Paper>
+  );
+};
+
+const Dashboard = ({ classes, loading, user }) => {
   if (loading) {
     return <CircularProgress className={classes.progress} size={50} />;
   }
 
   return (
     <div className={classes.root}>
       <Paper className={classes.paper} elevation={4}>
-        <Typography variant="headline" component="h3">
+        <Typography variant="headline" component="h3" align="center">
           Welcome to Callout
         </Typography>
-        <Typography component="p">
+        <Typography component="p" align="center">
           An open source availability and event response management system.
         </Typography>
       </Paper>
+      {/* TODO: Hide this when user has added their capabilities */}
+      <Paper className={classes.paper}>
+        <Typography component="p" align="center">
+          {/* TODO: update link t go edit profile page */}
+          You have no capabilities assigned for your user profile. <Link to="/groups">Click here to add capabilities</Link>
+        </Typography>
+      </Paper>
+      {displayRequests(classes, user)}
```

Comment:
just out of interest any reason you use a function here instead of defining a stateless component? I'm new to react.

Reply:
can do that but saw no justification if the component would not be used elsewhere

Reply:
You're 99% of the way there - with a few minor changes it becomes a component that is easier to test and reason about in the react view hierarchy. I'm not going to reject so you're free to merge without this if you still don't think it's worthwhile.
```
<DisplayRequests classes={classes} user={user} />
```